src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -3,16 +3,10 @@
 import java.util.Random;
 
 public class ColorSupplier {
-    private Random random;
-
-    public ColorSupplier() {
-        random = new Random();
-    }
-
     public String getRandomColor() {
         Color[] colors = Color.values();
+        Random random = new Random();
```

Comment:
Here we basically got to what we started with, this `random` variable should still be a field of `ColorSupplier` as Yevhenii said in his comment, to avoid its multiple instances creations, but he also pointed out that it should be initialized in the field right away and not in the constructor. 

I am sorry if the comments are confusing, I personally do not fully agree with the constructor detail of the review and do not see that big of a difference, but anyway, do not be afraid to tell us if you see a contradiction in the mentor comments, we try to avoid it, but sometimes they just happen. You can ask us both about it in the personal chat anytime and all of us can come up with the best possible approach if we discuss it together :)
We can still do that if you'd like, but for now, to avoid spending even more time on this, I'd rather agree with Yevhenii's comment, so let's do as he proposed